### ISSUE TITLE
Reputation functions to add before v1 mainnet launch

### ISSUE THREAD

**Issue by drwasho** [OpenBazaar/OpenBazaar-Client#840]
Unfinished sub-task from #472 
- [x] Display additional rating detail on the listing's `Reviews` tab
  - **OPTIONAL**
    - GUID of the node who rated the tx (active link to their store, if they're available)
    - Button to show JSON backend of the rating data (aka _transaction summary_)
  - **MANDATORY**
    - Display either a URL or button that links to the escrow transaction on Blockcypher
- [ ] ~~Display aggregate rating data on a user's page~~ `post v1 requirement`

**Comment by drwasho** [comment]
Server link https://github.com/OpenBazaar/OpenBazaar-Server/issues/270

**Comment by rmisio** [comment]
@drwasho @cpacia 
Re: Display aggregate rating data on a user's page

I think we would need this calculation done on the server. For the client to do it, the client would need to loop through all the items in a user's store and for each one make an API request to get all the reviews for that item.

So, consider a store with a 100 items, each containing an average of 500 reviews. Not only would you have 100 HTTP requests, but these payloads would be pretty large (what if an item has 5000 reviews..??).

Further-furthermore, this would be repeated each time the item page is loaded.

Any approach to implement this on the client would be incredibly inefficient and / or complex.

I've made a server side issue:
https://github.com/OpenBazaar/OpenBazaar-Server/issues/283

**Comment by drwasho** [comment]
Agreed.

**Comment by drwasho** [comment]
Ok, did a little research. This is an example of the response to a `get_ratings` call:

```
{
    "signature": "H9zpuirYQ82zzlDdQ13S4CiMj4o2aezT0JbCLLw5gx2gMHQ3elR/RlEyw9lXQTvXWkA7yh2U1SKeU/iug/qkLV8=",
    "tx_summary": {
      "delivery_time": "5",
      "buyer_key": "03d1a9b222c85aa306cf49f1b6ded6d86a1a5850359622aad04d9b4d60c73a77ae",
      "feedback": "5",
      "review": "I like dresses",
      "listing": "c8d1ff75d916aaa2bd0040ffa9b4552b9b720ee9",
      "customer_service": "5",
      "amount": 0.01,
      "address": "mfsYWTUTXwgrrEcVb72Z65GKf7nrMJqZxk",
      "quality": "5",
      "proof_of_tx": "c0AMge0ZELxt/tbxv/Wkwyn1dhWV/fmqpjgQ5ir9gpzn8NWIiTgoqnWWYqAIRMuNvzkhnCR+Ir/T59xumU5+Dg==",
      "description": "5"
    }
  }
```

What I think we should display is a button that links out to the `address` value via Blockcypher... perhaps something like this:

![image](https://cloud.githubusercontent.com/assets/3887694/13864619/c216e01a-ecee-11e5-8c5b-3a75059f7381.png)

@morebrownies this could definitely use your talents.

**Comment by jjeffryes** [comment]
Closed in #1189

**Comment by drwasho** [comment]
Great job Josh
